db/migrate/20160604175804_create_units.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUnits < ActiveRecord::Migration
+  def change
+    create_table :units do |t|
+      t.integer :building_id
+      t.string :name
+      t.integer :floor
```

Comment:
Maybe we will need another form of metadata around floors to describe the section of the building?

Are the building registries descriptive enough to provide an idea of how the buildings are laid out? Maybe can just link to the page for the time being instead of maintaining all the metadata until a feature requirement exists to warrant it? Or we can just use Building.description to paint a picture of what the floors mean for now.